kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveStores.java

```diff
@@ -26,6 +27,10 @@
  */
 public final class ResponsiveStores {
 
+  public static KeyValueBytesStoreSupplier keyValueStore(final ResponsiveKeyValueParams params) {
```

Comment:
nit: javadocs? Can pretty much just copy/paste from the others below but it would be good to clarify why/how this API is different, when to use it vs the others, etc

Reply:
yup! that was on my TODO list and I forgot 🙈